packages/dicomImageLoader/package.json

```diff
@@ -21,7 +21,12 @@
     "access": "public"
   },
   "scripts": {
-    "build": "echo 'build not implemented yet'",
+    "build:umd": "cross-env NODE_ENV=production webpack --config .webpack/webpack-dynamic-import.js",
```

Comment:
Please up above on the "main" import list as the:
module - the typescript file import name
exports - both the inline bundle and the external/JS bundle.

and then build both the inline bundle and the worker bundle.  If the worker is just a simple wrapper for the inline bundle, then it is easy to just ship a single build that has both - no custom compilation/setup required.